### ISSUE TITLE
DATA LOSS: 48-bit tiff saved with 24-bit colour depth

### ISSUE THREAD

**Issue by eduardo-heuvel** [linuxmint/pix#25]
Pix is able to correctly open and display 48-bit (16 bit per channel) colour depth tiff images and can then be edited (like rotating). Huray! However... on exiting Pix it asks if the changes to the images should be saved. On choosing Yes, Pix saves the images as tiff but only with 24-bit colour depth, without any warning or option to choose between 48 and 24 bit. The result is DATA LOSS!

**Comment by icarter09** [comment]
@eduardo-heuvel are you still experiencing this issue?

**Comment by eduardo-heuvel** [comment]
Yes, and things got even worse: it not only reduces the color depth but also downscales the image dramatically. If I have a 12MP 48-bit image it comes out as 1MP 24-bit.
